### DISCUSSION TITLE
Directly link from a commit to CI Summary

### DISCUSSION THREAD

**Discussion by ducaale** [community/community#52091] [Actions]
### Select Topic Area

Product Feedback

### Body

Currently, it takes two steps to navigate from a commit to the relevant CI summary. However, some frequently used features, including Job Summary and rerunning all failed jobs, are done from the summary page.

It would be nice if there were a direct link to the summary page in addition to individual jobs.

![Screenshot 2023-04-05 at 17 35 05](https://user-images.githubusercontent.com/16341131/230152240-61a56103-a1e5-46e0-96c2-a0ca77dc8aae.png)

![Screenshot 2023-04-05 at 17 37 31](https://user-images.githubusercontent.com/16341131/230152254-0a158894-a583-4743-b516-a68398922a57.png)

**Comment by filhodanuvem** [comment]
+1 on this 

It is quite often to use [$GITHUB_STEP_SUMMARY](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables) to add important information when a job fails. 

<img width="945" alt="Screenshot 2023-10-02 at 10 01 37" src="https://github.com/community/community/assets/94096/ed59015a-abb3-431e-9a5f-e4ab648f1406">

For example in my case I add a link to screenshots when my end to end tests fail. Going to the job logs is something that I would do if the summary doesn't give me information enough so as @ducaale suggested, a simple link to the summary page would help a lot 🙏🏾 . 

<img width="1359" alt="Screenshot 2023-10-02 at 10 01 55" src="https://github.com/community/community/assets/94096/f5766663-cfbc-4a3a-8c99-3ff9d1516b03">
